jerry-debugger/jerry_client.py

```diff
@@ -104,7 +104,7 @@ def do_next(self, args):
                     if res_type == result.END:
                         self.quit = True
                         return
-                    elif res_type == result.TEXT:
+                    if res_type == result.TEXT:
```

Comment:
Why do we need to change `elif` to  `if`? This applies to most of the cases.

Reply:
Pylint does not accepts elif here becasue of return in previous if statement:

************* Module jerry_client
jerry-debugger/jerry_client.py:104:20: R1705: Unnecessary "elif" after "return" (no-else-return)